.github/workflows/cmake.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+env:
+  ROOT_VERSION: root_v6.24.06.Linux-ubuntu20-x86_64-gcc9.3.tar.gz
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Comment:
I suggest to have an additional build running on Ubuntu 22 as well.

Maybe you want to test also a newer version of ROOT? Like `6.28/04`.

Reply:
So yes I think it's a good idea to test as much as possible. Nevertheless, we have 2,000 min available per month (as far as I understood the pricing). Given that currently, not that much development is going on this should be fine, but something to keep in mind.

Reply:
(Can we register a "private" runner, in case?)

Reply:
Yes, this is possible.